src/test/scala/com/comcast/xfinity/sirius/uberstore/segmented/SegmentedCompactorTest.scala

```diff
@@ -232,11 +249,34 @@ class SegmentedCompactorTest extends NiceTest with BeforeAndAfterAll {
     }
   }
 
+  it("should remove all delete events in appropriate segments that are older than COMPACTION_MAX_DELETE_AGE_HOURS") {
```

Comment:
Since we are considering the corner case in the code, can you create a test when the age is greater than the current time (ie nothing is deleted)?